front/public/locales/fr/translation.json

```diff
@@ -308,9 +308,9 @@
         "crossed-detectors": "{{count}} détecteurs",
         "crossed-detectors_one": "Un détecteur",
         "crossed-detectors_zero": "Aucun détecteur rencontré",
-        "crossed-track-ranges": "{{count}} tronçons",
-        "crossed-track-ranges_one": "Un tronçon",
-        "crossed-track-ranges_zero": "Aucun tronçon",
+        "crossed-track-ranges": "{{count}} sections",
+        "crossed-track-ranges_one": "Une section",
```

Comment:
Why do we use the number in letters here and in other trads, you used `1` ?

Reply:
I can put one/un everywhere